backend/package.json

```diff
@@ -15,10 +15,10 @@
   "dependencies": {
     "cors": "^2.8.5",
     "dotenv": "^16.3.1",
-    "express": "^4.18.2",
+    "express": "5.0.0-beta.3",
```

Comment:
question: using a beta version ? normal ?

Reply:
updated express version to latest stable